templates/webserver/webserver-service.yaml

```diff
@@ -25,3 +25,10 @@ spec:
       protocol: TCP
       port: {{ .Values.ports.airflowUI }}
       targetPort: {{ .Values.ports.airflowUI }}
+    {{- if .Values.authSidecar.enabled }}
+    - name: nginx-ui
+      protocol: TCP
+      port: 8084
+      targetPort: 8084
```

Comment:
```suggestion
```
Don't need to duplicate it, it defaults to `port`.

Reply:
this change has been added.